eservices-monitoring-exporter/src/main/scala/it/pagopa/interop/eservicesmonitoringexporter/Main.scala

```diff
@@ -60,6 +60,9 @@ object Main extends App {
   }
 
   def app(): Future[Unit] = resources()
+    .andThen { case Failure(e) =>
+      logger.error("Eservices monitoring exporter job got an error on configuration", e)
```

Comment:
As this may be related to other logic than configuration load
```suggestion
      logger.error("Unable to load resources", e)
```